screens/Profile.tsx

```diff
@@ -1,20 +1,254 @@
+        <TouchableOpacity
+          style={styles.errorButton}
+          onPress={() => navigation.navigate("Authentification" as never)}
+        >
+          Anmelden
+        </TouchableOpacity>
+      </View>
+    );
+  } else if (userNotFound) {
+    return (
+      <View style={styles.errorContainer}>
+        <Text style={styles.errorText}>User not found</Text>
+      </View>
+    );
+  } else
+    return (
+      <ScrollView style={styles.container}>
+        <Image source={posts[2].source} style={styles.avatar} />
+
+        <View style={styles.header}>
+          <View style={styles.userInfo}>
+            <Text style={styles.name}>{nickname}</Text>
+            <Text
+              style={{
+                fontSize: SIZES.medium,
+                color: COLORS.darkgray,
+                marginBottom: "3%",
+              }}
+            >
+              @{username}
+            </Text>
+            <Text style={{ marginBottom: 12, marginTop: 6 }}>{status}</Text>
+            <TouchableOpacity
+              style={styles.editButton}
+              onPress={() => console.log("Test")}
+            >
+              <Text style={{ fontSize: SIZES.medium }}>Profil bearbeiten</Text>
+            </TouchableOpacity>
+          </View>
+          <View style={styles.stats}>
+            <View style={styles.statItem}>
+              <Text style={styles.statValue}>{postsCount}</Text>
+              <Text>Beiträge</Text>
+            </View>
+            <TouchableOpacity
+              style={styles.statItem}
+              onPress={() => navigation.navigate("FollowerList", { type: 0 })}
+            >
+              <Text style={styles.statValue}>{followerCount}</Text>
+              <Text>Follower</Text>
+            </TouchableOpacity>
+            <TouchableOpacity
+              style={styles.statItem}
+              onPress={() => navigation.navigate("FollowerList", { type: 1 })}
+            >
+              <Text style={styles.statValue}>{followingCount}</Text>
+              <Text>Gefolgt</Text>
+            </TouchableOpacity>
+            <TouchableOpacity
+              style={styles.statItem}
+              onPress={() => navigation.navigate("FollowerList", { type: 2 })}
+            >
+              <Text style={styles.statValue}>2</Text>
+              <Text>Anfragen</Text>
+            </TouchableOpacity>
+          </View>
+        </View>
+        <Text
+          style={{
+            fontSize: SIZES.large,
+            fontWeight: "bold",
+            marginTop: "5%",
+            marginLeft: "4%",
+            marginBottom: "1%",
+          }}
+        >
+          Beiträge
+        </Text>
+        <View style={styles.posts}>
+          {posts.map((image, index) => (
+            <Post key={index} image={image.source} />
+          ))}
+        </View>
+      </ScrollView>
+    );
+};
+export default Profile;
 
 const styles = StyleSheet.create({
+  errorContainer: {
```

Comment:
auch hier eigenes stylesheet?

Reply:
ja, würde das aber generell einmal einheitlich festlegen und einordnen zusammen. Auf einem eigenen Branch und einheitlich für alle stylesheets in den tsx files. Bis dahin würde ich die styles erstmal in den jeweiligen files lassen, sonst wirds schnell unübersichtlich